### PR TITLE
Initialize the frontend part of the project

### PR DESCRIPTION
**Pull request by smddzcy**
## About

* **Related Issues:** https://github.com/bounswe/bounswe2017group5/issues/83

As the title says, I initialized the frontend part of the project using a UI kit: https://www.creative-tim.com/product/get-shit-done-kit

### Home
screencapture-file-users-smddzcy-bounswe2017group5-fe-index-html-1508712598892

### Login
screencapture-file-users-smddzcy-bounswe2017group5-fe-login-html-1508713012184

### Signup
screencapture-file-users-smddzcy-bounswe2017group5-fe-signup-html-1508713017259

### Groups
screencapture-file-users-smddzcy-bounswe2017group5-fe-groups-html-1508796659171

### Create Group
screencapture-file-users-smddzcy-bounswe2017group5-fe-create-group-html-1508797297938

### Group Details
screencapture-file-users-smddzcy-bounswe2017group5-fe-group-details-html-1508796648797

### Dropdown and Selected Checkbox Styles
screencapture-file-users-smddzcy-bounswe2017group5-fe-group-details-html-1508797352047

### CODE REVIEW

**Comment by smddzcy** [comment]
@triforce930 We should probably make use of the templating engine of Django to make it somewhat modular. I copied/pasted the navbar code 3 times ._. (even created a `template.html` file for future copy/paste...)

File: fe/assets/js/get-shit-done.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+/*!
+    
+ =========================================================
+ * Get Shit Done Kit - v1.4.1.0
```
Comment (karacasoft) [inline fe/assets/js/get-shit-done.js]: Very nice kit to use :+1:

**Comment by karacasoft** [comment]
> We should probably make use of the templating engine of Django...

We should, and we will, don't worry. 

Very good job, I liked how the pages were also designed for mobile by adding the meta tags. Simple but good stuff :+1:.
 **EXCEPT**, there's a shit ton of dependencies added all over the place. Are they really required? Can you describe the functionality of each dependency? I want to eliminate some of those, if possible.

I'll accept the request, right after you do this and add the other page that you were assigned to.

**Comment by smddzcy** [comment]
@triforce930 Dependencies:
`<link href="bootstrap3/css/bootstrap.css" rel="stylesheet" />` => Bootstrap CSS file, required
`<link href="assets/css/gsdk.css" rel="stylesheet" />` => CSS file of the kit, required
`<link href="assets/css/custom.css" rel="stylesheet" />` => Our custom CSS, required
`<link href="bootstrap3/css/font-awesome.css" rel="stylesheet">`=> Font Awesome, required for icons we'll use in the future
`<link href="http://fonts.googleapis.com/css?family=Grand+Hotel" rel="stylesheet" type="text/css">` => Our primary font
`<script src="jquery/jquery-1.10.2.js" type="text/javascript"></script>` => jQuery, required
`<script src="assets/js/jquery-ui-1.10.4.custom.min.js" type="text/javascript"></script>` => jQuery UI, will be required for interactions
`<script src="bootstrap3/js/bootstrap.js" type="text/javascript"></script>` => Bootstrap JS, required
```
<script src="assets/js/gsdk-checkbox.js"></script>
<script src="assets/js/gsdk-radio.js"></script>
<script src="assets/js/gsdk-bootstrapswitch.js"></script>
<script src="assets/js/get-shit-done.js"></script>
```
JS files of our kit, required
`<script src="assets/js/custom.js"></script>` => Our custom JS

I really don't care about our bundle size for now, maybe we can optimize it in the future 😄 It's the default dependencies of the UI kit.

**Comment by karacasoft** [comment]
All right, waiting for the other file... :+1:

**Comment by smddzcy** [comment]
@triforce930 All done, review please.

File: fe/groups.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+          </div>
+          <div class="group-info">
+            <div class="group-name">
+              <h4>Metal</h4>
+            </div>
+            <div class="group-member-count">
+              1M Members
+            </div>
+            <div class="group-description">
+              A community for all the metalheads... m/
```
Comment (karacasoft) [inline fe/groups.html:88]: \m/ pls. :V

File: fe/groups.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+              <input type="text" value="" class="form-control" placeholder="Search...">
+            </li>
+            <li>
+              <a href="javascript:void(0);">
+                <i class="fa fa-search"></i>
+              </a>
+            </li>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown">Samed Düzçay <b class="caret"></b></a>
+              <ul class="dropdown-menu">
```
Comment (karacasoft) [inline fe/groups.html:52]: I didn't like the options of this dropdown. But we can fix them while carrying them over to Django templates.

File: fe/assets/js/get-shit-done.js [hunk 4]
```diff
@@ -38,50 +38,50 @@ $(document).ready(function(){
...
     });
-    
+
     $('[data-toggle="gsdk-collapse"]').hover(
     function(){
             console.log('on hover');
             var thisdiv = $(this).attr("data-target");
 
             if(!$(this).hasClass('state-open')){
-                $(this).addClass('state-hover');
```
Comment (karacasoft) [inline fe/assets/js/get-shit-done.js]: Gotta remove all those whitespaces...

**Review by karacasoft — APPROVED** [review]
Good, I'll merge these in. Thank you so much :+1:

**Pull request merged**